pkg/config/config.go

```diff
@@ -29,6 +30,7 @@ const (
 	flagEnableDevLogging     = "enable-development-logging"
 	flagAWSAccountID         = "aws-account-id"
 	flagAWSRegion            = "aws-region"
+	flagAWSServiceEndpoint   = "aws-service-endpoint"
```

Comment:
If this flag should be a URL (as it seems it needs to be below), this should be called `aws-endpoint-url`.